packages/babel-helpers/src/helpers/applyDecs2311.ts

```diff
@@ -269,21 +276,6 @@ export default /* @no-mangle */ function applyDecs2311(
     var isSetter = kind === PROP_KIND.SETTER;
```

Comment:
Aside: Commenting on Line 273 `mapKey = name + "/" + isStatic;`:

It doesn't work when the computed key is a symbol. In the following example, Babel throws "Cannot convert a Symbol value to a string":

```js
var dec = x => x;

class C {
  @dec
  [Symbol.iterator]() {}
}
```
we can fix that in a new PR.